spec/views/idv/session_errors/throttled.html.erb_spec.rb

```diff
@@ -21,7 +21,10 @@
         t('idv.troubleshooting.options.contact_support', app_name: APP_NAME),
         href: MarketingSite.contact_url,
       )
-      expect(rendered).not_to have_link(href: return_to_sp_cancel_path)
+      expect(rendered).to have_link(
```

Comment:
This changed from `not_to have_link` to `to have_link`. Is that intended?

Reply:
hi @soniaconnolly  - yes. the  not_to have_link check was checking to see if there a cancel link wasn't present 🤔 .  not sure the history here, but there is no cancel button code on the page not even conditionally.  seemed like it could have said not to to have any random button listed there 😅 .  i did repurpose this line to check that the new failure_to_proof link is present though.